### ISSUE TITLE
Banner examples need screen reader text/label indicating the context

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#5890]
**Describe the issue. What is the expected and unexpected behavior?**
The banner component uses color to communicate information/context to users. This information/context is not given to screen reader users who cannot see the color. Since we allow users to add any different types of context (for example, Gov't systems uses these to indicate whether the information on a page is classified or not), we should at least provide an example in which the context is provided through a label or screen reader text.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
Go to the [banner documentation](https://www.patternfly.org/v4/components/banner) and notice that the colors can potentially indicate state such as success, danger, etc. 

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Example bug

**Comment by tlabaj** [comment]
See this PR on example of how to add screen reader text to indicate severity 
https://github.com/patternfly/patternfly-react/pull/6664

**Comment by nicolethoen** [comment]
Screen reader text should be fully configurable, but also have a default value set. It should also be wrapped using the `.pf-u-screen-reader` class so that it is only visible to screen readers.

This is a great opportunity to learn how to use a screen reader to test the update!

**Comment by thatblindgeye** [comment]
@jessiehuff  This would most likely require input from design and also a separate issue, but wanted to comment here since it's related. In addition to adding screen reader text, I'm wondering if it would be worth also indicating the severity/variant of a banner via icons as well as color for non-screen reader users who have a color deficiency? Screen shot below is using Chrome's "Emulate vision deficiencies" feature with the deuteranopia option enabled:

![Banner deuteranopia example](https://user-images.githubusercontent.com/70952936/167633590-ccbb0037-5ccb-4f85-94b4-554cd1a34c06.png)

**Comment by jessiehuff** [comment]
What do you think @mcarrano? I think Eric's comment above makes sense.

**Comment by mcarrano** [comment]
That's a really good catch @thatblindgeye . Would you mind opening a new design issue for this and we will take a look at it? Thanks.
